CsharpExtras/Extensions/CollectionExtension.cs

```diff
@@ -8,7 +8,7 @@ namespace CsharpExtras.Extensions
 {
     public static class CollectionExtension
     {
-        public static bool TryGetSingleton<TValueType>(this ICollection<TValueType> collection, out TValueType singleton)
+        public static bool TryGetSingleton<TValueType>(this ICollection<TValueType> collection, out TValueType? singleton) where TValueType : class

```

Comment:
Technically, this is a breaking change. Consider adding this change to the change log file.